expression/bench_test.go

```diff
@@ -611,6 +608,14 @@ func (s *testEvaluatorSuite) TestVectorizedBuiltinFunc(c *C) {
 			default:
 				c.Fatal(fmt.Sprintf("evalType=%v is not supported", testCase.retEvalType))
 			}
+
+			// check warnings
+			totalWarns := ctx.GetSessionVars().StmtCtx.WarningCount()
+			c.Assert(2*vecWarnCnt, Equals, totalWarns)
+			warns := ctx.GetSessionVars().StmtCtx.GetWarnings()
+			for i := 0; i < int(vecWarnCnt); i++ {
+				c.Assert(warns[i].Err.Error(), Equals, warns[i+int(vecWarnCnt)].Err.Error())
```

Comment:
Will `Error()` return the error stack message?
If so, will it be better to use `c.Assert(ErrorEqual(err1, err2), IsTrue)` here?

Reply:
You're right.
It is fixed.